extension/bootstrap.js

```diff
@@ -78,20 +74,18 @@ var PersonasPlusBootstrapAddon = {
         Cu.unload("resource://personas/modules/personas.js");
         this.requestRemovePrerequisites(data);
         if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
-            // modules that come with Firefox
-            Cu.import("resource://gre/modules/XPCOMUtils.jsm");
-            // LightweightThemeManager may not be not available (Firefox < 3.6 or Thunderbird)
-            try {
-                Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
-            } catch (e) {
-                LightweightThemeManager = null;
-            }
-            try {
-                LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
-            } catch (e) {
-                try {
-                    LightweightThemeManager.currentTheme = null;
-                } catch (e) {}
+            if(LightweightThemeManager.currentTheme){
+                var prompt = Components.classes["@mozilla.org/embedcomp/prompt-service;1"].getService(Components.interfaces.nsIPromptService);
+                if (prompt.confirm(null, "Personas Plus", "Would you like to remove the current theme?")) {
+                    try {
+                        Cu.import("resource://gre/modules/LightweightThemeManager.jsm");
+                        LightweightThemeManager.forgetUsedTheme(LightweightThemeManager.currentTheme.id);
```

Comment:
> BTW, LightweightThemeManager.forgetUsedTheme does not throw error, if the theme id is not available. It just silently fails.

OK, good to know.

But, if you explicitly ask the user if the current theme should be uninstalled (and that theme happens to be _Groovy Blue_), you should not remove if the user answered _no_.

That's why I suggested to call this routine only if _Groovy Blue_ is not the current theme.